### PR TITLE
Add nerdfont icons for .bicep & .bicepparam files

### PR DESCRIPTION
**Pull request by scottmckendry**
- **PR Description**
As the title says, I've added the v3 nerd font icons for [Bicep](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep). 

- **Please check if the PR fulfills these requirements**

* [x] Cheatsheets are up-to-date (run `go generate ./...`)
* [x] Code has been formatted (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#code-formatting))
* [x] Tests have been added/updated (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/integration/README.md) for the integration test guide)
* [x] Text is internationalised (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#internationalisation))
* [x] Docs (specifically `docs/Config.md`) have been updated if necessary
* [x] You've read through your own file changes for silly mistakes etc

### CODE REVIEW

**Comment by mark2185** [comment]
@scottmckendry could you please rebase this and resolve the conflicts, if you're up for it?

**Comment by scottmckendry** [comment]
@mark2185 done - I'm not sure about colours, however. Bicep doesn't have very distinct branding so I've sort of picked these at random. 

Happy for these to be updated to whatever. I have no preference at all when it comes to colours, as long as it's easy to see the difference between the two file types (given they share an icon).

File: pkg/gui/presentation/icons/file_icons.go [hunk 1]
```diff
@@ -326,7 +328,12 @@ func patchFileIconsForNerdFontsV2() {
 	extIconMap[".vue"] = IconProperties{Icon: "\ufd42", Color: 113}     // ﵂
 }
 
-func IconForFile(name string, isSubmodule bool, isLinkedWorktree bool, isDirectory bool) IconProperties {
```
Comment (mark2185) [inline pkg/gui/presentation/icons/file_icons.go:329]: Is this something the linter changed or?

Comment (scottmckendry) [reply pkg/gui/presentation/icons/file_icons.go]: I have fixed this now

**Comment by jesseduffield** [comment]
Thanks @scottmckendry !

**Pull request merged**